features/column-pseudo.yml

```
@@ -0,0 +1,4 @@
+name: ::column pseudo-element
+description: The `::column` CSS pseudo-element represents the individual columns in a multi-column container and can be styled with scroll-snap properties and have associated `::scroll-marker` pseudo-elements.
+spec: https://drafts.csswg.org/css-multicol-2/#column-pseudo
+group: multi-column
```

```suggestion
group:
  - multi-column
  - scrolling
```

So that the feature is also grouped with the other scroll-button/scroll-marker features.